main.cc

```diff
@@ -447,6 +449,18 @@ int main(int ac, char** av) {
         return 0;
     }
 
+    try {
+        build_id = get_build_id();
+    } catch (...) {
+        fmt::print("Could not get build_id: {}\n", std::current_exception());
+        build_id = "<unknown>";
```

Comment:
I don't think we should run if we failed to print a build_id. If this fails it means that there is something horribly broken with out build (which there is).

It is obviously OK to print a better error message and exit() instead of failing an assert.

Reply:
I beg to differ.  Not having the build_id is not a critical error we should crash on.  As seen it could be just a bug on our side, or something that changed in toolchain.

Reply:
Please, use a `startlog` logger instead of printing to `stdout`